packages/react-scripts/config/webpack.config.js

```diff
@@ -62,6 +62,9 @@ const imageInlineSizeLimit = parseInt(
 // Check if TypeScript is setup
 const useTypeScript = fs.existsSync(paths.appTsConfig);
 
+// Check disable incremental build in fork-ts-checker-webpack-plugin
+const useTypeScriptIncrementalBuild = !process.env.DISABLE_TS_INCREMENTAL_BUILD || true;
```

Comment:
The `useTypeScriptIncrementalBuild ` is always be `true`. And `DISABLE_TS_INCREMENTAL_BUILD` is string.

https://webpack.js.org/plugins/define-plugin/#usage
 > If the value isn't a string, it will be stringified (including functions).

Reply:
Thx for checking!✨
https://github.com/facebook/create-react-app/pull/10365/commits/ccbb0a5fdc6d4fe6456323178dda1a7c0c3069dd: fixed modify boolean check logic 🙇‍♀️